ci/ubuntu/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Call the setup script
+echo "Setting up the environment..."
+./setup_env.sh
+echo "Environment setup completed."
+
+# Array to store build statuses
+build_statuses=()
+
+# Build civkit-cli
+echo "Building civkit-cli..."
+CLI_STATUS=$(./build_civkit-cli.sh)
+build_statuses+=("$CLI_STATUS")
+
+# Build civkitd
+echo "Building civkitd..."
+CIVKITD_STATUS=$(./build_civkitd.sh)
+build_statuses+=("$CIVKITD_STATUS")
+
+# Build civkit-sample
+echo "Building civkit-sample..."
+SAMPLE_STATUS=$(./build_civkit-sample.sh)
+build_statuses+=("$SAMPLE_STATUS")
+
```

Comment:
This sounds all good to me, we have at the top `run.sh`:
- calling `setup_env.sh` to configure the good rust env
- calling `build_civkit-cli.sh` for `civkit-cli`
- calling `build_civkitd.sh` for `civkitd`
- calling `build_civkit-sample.sh` for `civkit-sample`

I think this is still missing a `.github/workflows/build.yml` to do the linkage with Github Actions.

Reply:
This code is for running such tests in a local setup. 

I have committed a change to run the binary tests for the resources civkitd, cli and sample) on ubuntu in this branch. the setup for a local run test vs github actions is slightly different. Rather then make life complicated, I propose we have the option for local runs with GA actions configured to automate this. 

Feel free to review here the output of a GA test here
https://github.com/dapsavoie/civkit-node/actions/runs/5475581696